2dr.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url(https://fonts.googleapis.com/css?family=Merriweather:900|Roboto)
+}
+
+@font-face{
+    font-family: 'Merriweather';
+    src: url(https://fonts.googleapis.com/css?family=Merriweather:900|Roboto)
+}
+body{
+    width: 625px;
+    color: white;
+    
+    
+}
+h1{
+    font-family: 'Merriweather';
+    font-weight: 900;
+    border-bottom: 15px;
+    border-top: 30px solid transparent;
+    margin: unset
+
+    
+    
+}
+p {
+    font-family: 'Roboto', serif;
+    font-weight: 400;
+    border-bottom: 20px solid transparent;
+    width: 500px
+}
+.zuta{
+    background-image: url("file:///C:/Users/nikol/Desktop/slike/žuta.jpg");
```

Comment:
This is your local filepath. Images won't show on my computer till they are not within the project folder and included from there.